enterprise/api/v1/views.py

```diff
@@ -1483,3 +1484,51 @@ def update(self, request, *args, **kwargs):
     )
     def destroy(self, request, *args, **kwargs):
         return super().destroy(request, *args, **kwargs)
+
+    @action(methods=['post'], detail=True, url_path='link-user')
+    def link_user(self, request, pk=None):
+        """
+        Post
+        Links user using enterprise_customer_key
+        /enterprise/api/enterprise_user_link/{enterprise_customer_key}/link-user
```

Comment:
nit: The URL is now different with this as an action under `EnterpriseCustomerInviteKeyViewSet`. Should it be?

```
/enterprise/api/v1/enterprise-customer-invite-key/<enterprise_customer_invite_key_uuid>/link-user/
```